### PR TITLE
Add middleware for redirects in app-root

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
app-root

## Linked Issue and/or Talk Post
[Spreadsheet](https://docs.google.com/spreadsheets/d/1zA76BiSxMN7X4IadKFRzyRpsX5JnR46fGJKypnSkXio/edit?gid=317530152) of re-routing

## Describe your changes
- Add a [middleware.js](https://nextjs.org/docs/app/building-your-application/routing/middleware) file to handle legacy PFE `/about` and `/get-involved` paths. When the About Zooniverse pages are launched on FEM, users might have the legacy links saved, so we want to redirect them to the correct info.
- ZooHeader includes a link to `/get-involved`. There's no root Get Involved page in FEM, so I added a redirect to `/get-involved/volunteer`.

## How to Review

- Looking at the conditional statements in the middleware, point your browser to each url and see that you're redirected to the correct page.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)

### CODE REVIEW

**Comment by goplayoutside3** [comment]
@zwolf I'm including you as a reviewer just for awareness, but @mcbouslog please review this PR locally. This PR can be merged prior to launching new FEM pages.

File: packages/app-root/src/middleware.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// Note that when internationalization is introduced to app-root,
+// this file may need refactoring to handle locale in pathname
+
+import { NextResponse } from 'next/server'
+
+export function middleware(req) {
+  /*
+    Redirect legacy PFE /about and /get-involved paths to new FEM paths
+  */
+  if (req.nextUrl.pathname.startsWith('/about/acknowledgments')) {
```
Comment (mcbouslog) [inline packages/app-root/src/middleware.js:10]: Currently per PFE, https://www.zooniverse.org/about/acknowledgments -> "Not found", and https://www.zooniverse.org/about/acknowledgements -> loads. British English includes the extra e, which matches PFE, which I think we should match here if someone has bookmarked it?

**Review by mcbouslog — APPROVED** [review]
LGTM, confirmed redirects work as expected 👍 

See inline comment regarding acknowledgments vs acknowledgements and existing PFE route https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/router.jsx#L121 .

File: packages/app-root/src/middleware.js (hunk 1 above)
Comment (goplayoutside3) [reply packages/app-root/src/middleware.js:10]: Ah good point, I'll add that redirect before merging!

**Pull request merged**